controllers/feeds/getFeedsV2.js

```diff
@@ -182,7 +192,7 @@ module.exports = async (req, res) => {
             item.is_self =
               item.actor.id === req.userId || item.actor.id === myAnonymousUser?.user_id;
             let newItem = await modifyFeedIsFollowingTarget(item, req.userId);
-            newItem = await modifyAnonimityPost(newItem);
+            newItem = await modifyAnonimityPost(newItem, isBlurredPost);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `isBlurredPost` as a parameter to the `modifyAnonimityPost` function is correct. However, ensure that the `modifyAnonimityPost` function has been updated to handle this new parameter correctly and that it doesn't break existing calls to this function elsewhere in the codebase.
```diff
- newItem = await modifyAnonimityPost(newItem);
+ newItem = await modifyAnonimityPost(newItem, isBlurredPost);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->